spec/lib/zendesk_apps_tools/cache_spec.rb

```diff
@@ -6,12 +6,13 @@
 
 describe ZendeskAppsTools::Cache do
   context 'with a local cache file' do
-    let(:cache) { ZendeskAppsTools::Cache.new(path: tmpdir) }
-    let(:tmpdir) { Dir.mktmpdir }
+    let(:tmpdir)   { Dir.mktmpdir }
+    let(:cache)    { ZendeskAppsTools::Cache.new(path: tmpdir) }
+    let(:zat_file) { File.join(tmpdir, '.zat') }
 
     before do
       content = JSON.dump(subdomain: 'under-the-domain', username: 'Roger', app_id: 12)
-      File.write(File.join(tmpdir, '.zat'), content, mode: 'w')
+      File.write(zat_file, content, mode: 'w')
```

Comment:
Rather than actually writing to the file system for this test, would it be sufficient (and beneficial?) to instead write to an object like a [StringIO](https://ruby-doc.org/stdlib-2.5.1/libdoc/stringio/rdoc/StringIO.html) here?